src/plugins/ncr/components/labels/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
@rsumilang i'd probably call this node-labels to it's clear it's not something like form labels or generic labels.